docs/specification.yml

```diff
@@ -0,0 +1,616 @@
+openapi: 3.0.0
+info:
+  description: |-
+    This is a definition of CSC Metadata Submitter API.
+
+  version: 1.0.0
+  title: CSC Metadata Submitter API
+  license:
+    name: License Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+tags:
+  - name: Submission
+    description: Endpoints used in submission
+  - name: Query
+    description: Endpoints used for data retrieval
+  - name: Manage
+    description: Endpoints used for managing data
+paths:
+  /submit:
```

Comment:
Current version of `/submit` -endpoint allows insertion of multiple objects types at a time (e.g. study+sample+experiment), so response with single json accessionId doesn't reflect to what happened in db. Probably answer with json containing accessionId for each added object?

Reply:
good catch. changed to `ObjectsCreated` so that it also returns the schemas of the objects that were added.